app/services/campaign_page_builder.rb

```diff
@@ -26,9 +26,15 @@ def page
   end
 
   def create_plugins
-    page.liquid_layout.partial_refs.each do |partial, ref|
-      plugin_name = LiquidPartial.find_by(title: partial).plugin_name
-      Plugins.create_for_page(plugin_name, page, ref)
+    if default
+      Plugins.registered.each do |plugin|
+        Plugins.basic_create_for_page(plugin, page)
+      end
```

Comment:
I don't understand what you're going for here. Why would we want to create one of each type of plugin instead of the plugins the page actually uses? And why would an ENV variable control that override?